src/frontend/components/SchemaForm/_RenderFormInput.tsx

```diff
@@ -70,10 +71,37 @@ export function RenderFormInput(props: IRenderFormInputProps) {
 
   switch (type) {
     case "email":
-    case "password":
     case "url":
     case "color":
       return <FormInput type={type} {...formProps} />;
+    case "password":
+    return (
+        <FormInput
+          type={isPasswordReveal.isOn ? "text" : "password"}
+          {...formProps}
+          rightActions={
+            formProps.input.value
```

Comment:
As mentioned above the logic of the rightAction always be present and should be based on the `isPasswordReveal.isOn` so we should be having something like

```js
rightActions: [showPassword.isOn ? {
systemIcon: 'EyeOff',
 action: showPassword.toggle,
  label: msg`Hide Password`,
}: {
systemIcon: 'Eye',
 action: showPassword.toggle,
  label: msg`Show Password`,
}]
```